tests/test_ext_commands.py

```diff
@@ -260,6 +261,21 @@ def test_sanitize_repo_raises_plug_error_if_file_list_doesnt_exist(
         )
 
 
+def test_sanitize_repo_raises_invalid_repo_if_invalid_repo(
+    sanitizer_config, fake_repo
+):
+    target_branch = "student-version"
+    git_path = fake_repo.path / ".git"
+    shutil.rmtree(git_path, ignore_errors=True)
+
+    with pytest.raises(git.InvalidGitRepositoryError):
+        execute_sanitize_repo(
+            f"--file-list {fake_repo.file_list_path} "
+            f"--repo-root {fake_repo.path} "
+            f"--target-branch {target_branch} "
+        )
```

Comment:
Sorry, I should have mentioned, we should convert this to a `plug.PlugError` with a clear error message. The test should look something like this:

```suggestion
    with pytest.raises(plug.PlugError) as exc_info:
        execute_sanitize_repo(
            f"--file-list {fake_repo.file_list_path} "
            f"--repo-root {fake_repo.path} "
            f"--target-branch {target_branch} "
        )
    
    assert f"Not a git repository: '{fake_repo.path}'" in str(exc_info.value)
```